tests/unit/lib/Any_unittest.cc

```diff
@@ -691,3 +691,69 @@ TEST(Any, lazy_cloneable_visitable)
 	EXPECT_FALSE(any.visitable());
 	EXPECT_THROW(any.visit(nullptr), std::logic_error);
 }
+
+TEST(AnyParameterProperties, old_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_EQ(params.get_description(), "No description given");
+	EXPECT_EQ(
+	    params.get_model_selection(),
+	    EModelSelectionAvailability::MS_NOT_AVAILABLE);
+	EXPECT_EQ(
+	    params.get_gradient(), EGradientAvailability::GRADIENT_NOT_AVAILABLE);
+	EXPECT_FALSE(params.get_model());
+}
+
+TEST(AnyParameterProperties, new_api_default)
+{
+	AnyParameterProperties params = AnyParameterProperties();
+
+	EXPECT_EQ(params.get_description(), "No description given");
+	EXPECT_FALSE(params.is_property(
```

Comment:
imagine we get more properties ... this test then needs to be touched which makes maintenance noisy.
What about just asserting that no parameters are set. i.e. the mask is zero everywhere

Reply:
OK! So you mean have three tests, one for each parameter? The only thing that feels like is missing is an enum for zero, since we can't compare the `enum class` to an `int`, i.e. 0.

Reply:
well I guess 0 and `ParameterProperties()` are the same, so never mind!

Reply:
I would just write a test that ensures that the default parameter properties are 0